### PR TITLE
Update version rc3 24117 00

### PR DESCRIPTION
**Pull request by StephenBonikowsky**
Two commits...
 The first one just updates our package versions to rc3-24117-00
 The second one is to port PR dotnet/corefx#8565 which required an update to the buildtool version and a couple packages to be added to the project.json for System.ServiceModel.Primitives.

### CODE REVIEW

**Comment by StephenBonikowsky** [comment]
@dotnet-bot test all outerloop please

**Comment by roncain** [comment]
LGTM

File: BuildToolsVersion.txt [hunk 1]
```diff
@@ -1 +1 @@
-1.0.25-prerelease-00420-01
\ No newline at end of file
+1.0.25-prerelease-00424-03
```
Comment (roncain) [inline BuildToolsVersion.txt]: CoreFx is now up to 00425-03.  We should probably update to match.  But that might be handled by #1213 if it gets merged first

Comment (StephenBonikowsky) [inline BuildToolsVersion.txt]: version 00425-03 causes build errors, I am going to merge 00424-03 which works and then investigate the errors for 425-03.

**Comment by StephenBonikowsky** [comment]
Ubuntu build failure is some Jenkins issue, merging.

**Pull request merged**